### DISCUSSION TITLE
Map Feature

### DISCUSSION THREAD

**Discussion by zainfathoni** [kawalcovid19/wargabantuwarga.com#32] [Ideas]
# Background

[kitajaga.co](https://kitajaga.co/) managed to get a pretty significant tractions of [digitizing the white flags movement in Malaysia](https://www.thestar.com.my/tech/tech-news/2021/07/07/want-to-help-those-in-need-search-for-white-flags-in-your-area-with-the-kita-jaga-malaysia-website). We are looking forward to replicate it for our use cases in Indonesia.

## Problem Statement

TBD

## Proposed Solution

TBD

### Product Requirements

- ...
- ...

### Implementation Plan

- [ ] ...
- [ ] ...
- [ ] ...

**Comment by zainfathoni** [comment]
FYI, when I was experimenting with #46, I bumped into this Map feature in [Netlify CMS' Kitchen Sink](https://cms-demo.netlify.com/#/collections/kitchenSink/entries/a-big-entry-with-all-the-things).

**Comment by rasisbuldan** [comment]
+1 on this, would be super helpful and align with **_warga bantu warga_** theme. If implementing map is a long shot, do we have to consider another alternative implementation, like some sort of list with subdistrict (kecamatan) or zip codes?

**Reply by baraeb92** [reply]
We do have plan that currently being discussed about having a list of resource center/help basecamp for every cities.

Where people can come to donate resources (money, food, medicine) and of which people who need all of those can come and contact the help center in every city to get the resource they need during the lockdown or isoman. 

So this place in every city will act as a distribution center. 

The will be the 1st iteration, before going big on maps. So we are still working on managing the logistic. 

Also urundaya covid already has a MVP for a map for now, so we can build on top of it's idea as well.
